websocket/_core.py

```diff
@@ -20,8 +20,15 @@
 """
 from __future__ import print_function
 
+try:
+	import six	#try to import an installed library
+except ImportError:
+	try:
+		from ..websocket import six	#try to import six.py at ../websocket
+	except ValueError:
+		print('Please install six')
+		exit()
```

Comment:
so should i just leave it at the print line, because doing so would result in an error later on as soon as the name 'six' is used?